moto/sagemaker/models.py

```diff
@@ -205,6 +205,7 @@ def __init__(
         client_request_token: str,
     ):
         self.pipeline_execution_arn = pipeline_execution_arn
+        self.arn = self.pipeline_execution_arn
```

Comment:
I realize that it's quite a lot of work to rewrite these - but I think we should all settle on `self.arn`, and get rid of the `self.pipeline_execution_arn`. Having two properties to point to the same value is going to be confusing, and can lead to drift.

(Same for all the other models..)